src/Indice.Features.Cases.AspNetCore/Data/Config/DbCheckpointTypeConfiguration.cs

```diff
@@ -13,9 +13,9 @@ internal class DbCheckpointTypeConfiguration : IEntityTypeConfiguration<DbCheckp
             builder
                 .HasKey(p => p.Id);
             builder
-                .HasIndex(p => p.Name);
+                .HasIndex(p => new {  p.Id, p.Code });
```

Comment:
`p.CaseTypeId, p.Code`

Θελουμε μοναδικότητα στις εγγραφές στην βάση όσον αφορά το casetyp Id και το Code.

Δηλαδη to code `Rejected` να υπάρχει μία φορά για το case Type Id `1234`. 

Όπως το έχεις τώρα είναι τσαμπα index γιατί το Id είναι πάντα unique (είναι το PK)